cmd/kured/main.go

```diff
@@ -266,7 +310,14 @@ func commandReboot(nodeID string) {
 	}
 
 	// Relies on hostPID:true and privileged:true to enter host mount space
-	rebootCmd := newCommand("/usr/bin/nsenter", "-m/proc/1/ns/mnt", "/bin/systemctl", "reboot")
+	pid := getPIDtoRunCmds()
```

Comment:
Instead of running this, why don't we accept a parameter for the commandReboot to take the reboot command as a parameter instead?

The commandReboot would almost not change, it would just a question of how to pass the right value to it.

Reply:
ok!

Reply:
I think that keeping the logic of the reboot within the reboot function commandReboot is clearer.